### ISSUE TITLE
Oracle view DDL

### ISSUE THREAD

**Issue by mkgrgis** [dbeaver/dbeaver#8765]
[x:oracle]
#### System information: 
- Operating system (distribution) and version
Linux Mint, Ubuntu 20.04
- DBeaver version
7.0.5
- Additional extensions
No
#### Connection specification:
- Database name and version
Oracle 10.0
- Driver name
from maver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
No

I have "true" in flag "case-sensetive names" for metadata, but views usually has "UPPERCASE" names. Columns usually has "CaMelCase"  names.

#### Describe the problem you're observing:
1. Open any view - > generate DLL
-> nullPointerException

2. If we call view - > properties -> declaration
-> unstable, but for many views there is only comment "--empty source"

**Comment by uslss** [comment]
hi @mkgrgis ,
unfortunately, cannot reproduce. is there any info in logs? is issue reproducible against 7.0.5?
thanks in advance for response

**Comment by mkgrgis** [comment]
Yes, my new test for 7.0.5. It was update.
```
﻿{
"SELECT * FROM V$VERSION": [
	{
		"BANNER" : "Oracle Database 10g Release 10.2.0.4.0 - 64bit Production"
	},
	{
		"BANNER" : "PL\/SQL Release 10.2.0.4.0 - Production"
	},
	{
		"BANNER" : "CORE\t10.2.0.4.0\tProduction"
	},
	{
		"BANNER" : "TNS for 64-bit Linux: Version 10.2.0.4.0 - Production"
	},
	{
		"BANNER" : "NLSRTL Version 10.2.0.4.0 - Production"
	}
]}
```
![1](https://user-images.githubusercontent.com/41448637/82572538-27e16b80-9b8d-11ea-908a-ee17c28e899e.png)
![2](https://user-images.githubusercontent.com/41448637/82572540-29129880-9b8d-11ea-9582-f508c1534ea5.png)
![3](https://user-images.githubusercontent.com/41448637/82572542-29ab2f00-9b8d-11ea-9413-859b8d270dfa.png)

**Comment by mkgrgis** [comment]
It seems there is \t and whitespaces in the source of the view.

**Comment by mkgrgis** [comment]
Error with "--empty source" is unstable for the same stable connection during any hours. But the error with DDL generate is stable.

**Comment by mkgrgis** [comment]
Solution for situation with "--Empty source" is updating (press F5) of source of view. Real "--Empty source" is very rare in Oracle. I think, if there is "--Empty source" it can signalize to once later refresh source text area data.

nullPointerException for view's DDL still exist.

**Comment by yonisade** [comment]
Happens to me as well on 7.0.5 with Oracle 11.2.0.4

I wanted to generate DDL to multiple views at once but got the same error.
My workaround was to select the views, right click,  "Compare/Migrate","Simple Compare", "Next", "Compare", and then right click again on views and "Tools->generate DDL"

**Comment by galendboyer** [comment]
On DBeaver, 7.1.0, I also had NULL pointer exception when extracting SQL for views from Oracle.  I followed yonisade's path and that worked.  Let me know what I can give you or track down to help you solve the bug.

**Comment by serge-rider** [comment]
@galendboyer please check error log (https://github.com/dbeaver/dbeaver/wiki/Log-files)
Unfortunately  I can't reproduce this on my side.

**Comment by mkgrgis** [comment]
New bug for Oracle view's DDL. There is comments for columns and main object in DDL for Oracle table, but there is no comments for Oracle view.
Steps to reproduce:
1. Create a table with comment and comments for columns
2. Create a view with comment and comments for columns
3. Compare DDL for this objects.

**Comment by serge-rider** [comment]
Fixed in #9348

**Comment by kseniiaguzeeva** [comment]
Verified.
1. Comments are displayed in DDL in Generate SQL>DDL
2. NPE doesn't appear after pressing Generate SQL>DDL for Oracle view and View Declaration tab
